src/Microsoft.VisualStudio.Web.CodeGeneration.Design/Shared.proj

```diff
@@ -0,0 +1,17 @@
+<Project>
```

Comment:
Nit: it's MSBuild convention use the `.props` extension instead of .`proj`. This file is only intended to be used via Import, and not invoked directly.